### PR TITLE
Prototype Hardsuit

### PR DESCRIPTION
**Pull request by uraniummeltdown**
rdhardsuit
Adds the Prototype Hardsuit from TG, a bombsuit+hardsuit with some fire protection for the RD. Comes with a diagnostic HUD, research scanner.
Also it's supposed to function like the tachyon-doppler array showing you the strength of bombs but for some reason I can't get that to work, would like some help getting it to work (no runtimes or anything so far)
All hardsuits now show a little message when in range of an EMP, showing you if it's light or strong.

:cl:
add: Added Prototype Hardsuit for the RD, in a suit storage unit in his office.
add: All hardsuits show a message when in range of an EMP, showing the strength of the EMP.
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
Does the RD need more power?

**Comment by uraniummeltdown** [comment]
This is a suit to protect the RD from the dangers of his work environment, Toxins and the Test Lab frequently blow up or are on fire, the diagHUD is useful in working with mechs and borgs, research scanner for RnD. The RD has access to EVA anyway.

**Comment by DesolateG** [comment]
I don't think this is needed for the RD.

File: code/modules/clothing/spacesuits/rig.dm [hunk 1]
```diff
@@ -473,6 +488,51 @@
...
+/obj/item/clothing/head/helmet/space/rig/rd
+	name = "prototype hardsuit helmet"
+	desc = "A prototype helmet designed for research in a hazardous, low pressure environment. Scientific data flashes across the visor."
+	icon_state = "rig0-rd"
+	item_color = "rd"
+	burn_state = FIRE_PROOF
+	unacidable = 1
+	var/onboard_hud_enabled = 0 //stops conflicts with another diag HUD
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	armor = list(melee = 30, bullet = 5, laser = 10, energy = 5, bomb = 100, bio = 100, rad = 60)
```
Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/rig.dm]: lower the melee to be in line with all our other hardsuit's melee armor; we never ported over the "buff to all hardsuit melee stats"; this shouldn't get it either.

**Comment by Fox-McCloud** [comment]
Seems alright, as long as you change the melee armor; ideally, every single head of staff should start out with a hardsuit, with teh exception of maybe the HoP.

**Comment by DesolateG** [comment]
The RD starts out with telereactive armor, why does he need a hard suit? Also, HoS and CMO don't -start out- with hard suits, they simply have access to them.

**Comment by Anoonki** [comment]
> every single head of staff should start out with a hardsuit, with teh exception of maybe the HoP.

Can we please have the HoS hardsuit actually on the map? It's actually in the code fully functionally afaik

**Comment by SamHPurp** [comment]
Realistically HoS shouldn't leave station, but if the air/structural integrity is compromised then fine.

RD however... why not use regular bomb suits etc for this...?

**Comment by jughu** [comment]
Bombs make breaaches bomb suit wont protect you from those hardsuits do plus 100% to not gib your body ideally the hardsuit to disarm bombs

**Comment by DesolateG** [comment]
Bomb shit protects you from gibbing

**Comment by jughu** [comment]
@desolateg i didnt word this correctly

Bomb armor is chance based
Bomb suits (bio ones) have 25% of not gibbing you and reducing a little explosion damage
Bomb hardsuit (this pr) has 100% of not gibbing even if the bomb on the same tile as you it will destroy your hardsuit sure but not gib you

**Comment by Aurorablade** [comment]
@uraniummeltdown if you wanted to go a step further you might as well just port over all of TGs hardsuit stuff to equal things out.

**Comment by Doukhan** [comment]
RD now gets spaceproofing ontop of his infinite baton, when are hardsuits going to be slowed down indoors?

**Comment by optimumtact** [comment]
Do not follow our path of hardsuit creep

**Comment by pinatacolada** [comment]
P H U C K    N O

RD has absolutely 0 reason to be in space/vented areas.

His toxins lab blew up, tough luck, engineering is fixing it.
His scientist is in the breach in toxins, tough luck, medical (paramedic if they ever did their jerbs) is recovering him
telescientist divided by zero and spaced himself with the teleporter, tough luck, medical, pod pilot and maybe the mechanic are looking for him.

By your logic, the RD and the entire science department should round start with laser guns because "xenobio might make life in the hall"


And these are not even mechanical/balance reasons of why him shouldn't have a space suit.

**Comment by Spartan6** [comment]
RD doesn't need this.
At all.

**Comment by DesolateG** [comment]
@Spartan6 if RD had much stricter SoP and no chance at antag, sure why not... but not in their current state, no.

**Comment by pinatacolada** [comment]
i'm honestly not even that phased by the antag prospect, I'm more concerned in making sure the RD is not a jack of all trades. If his lab blew up he should be dependent on engineering in fixing it. If his mentally defective coworker spaced himself he should be dependent on medical or a pod pilot on fetching him

**Comment by uraniummeltdown** [comment]
@pinatacolada He can already just grab a suit from EVA
@Fox-McCloud will do

**Comment by Anasari** [comment]
Science don't need yet another buff.

**Comment by davidchan13** [comment]
Just because CE gets a hardsuit doesn't mean every other head needs a hardsuit. And as citrus said, one of the many mistakes and power creep tg made was adding hardsuits to CMO/HOS/RD while mysteriously passing over the HoP. Heads rarely need a space suit and giving them crap like this just encourages them to piss off their job or go hide in space when they should be doing something useful on the station.

**Comment by taukausanake** [comment]
Can you make this researchable? It looks like a nice addition if it fully works but it REALLY doesn't need to belong to the RD. The only way I can see adding this to the code is making it an R&D thing but that only slows its acquisition. Maybe it would go better somewhere else but I can't think of anywhere unless the derelict station has a science area in it but that ends up limiting it to Mechanic and miners who get a pod from the asteroid. And Mechanic already gets the whiteship

**Comment by Markolie** [comment]
I don't think we should have researchable hardsuits. Their access needs to be limited.

**Comment by Kyep** [comment]
I'm OK with this being merged, or not merged. Either way, it is not a big deal, and doesn't deserve to be as controversial as it is.

RD already has access to EVA suits. A recolored, bomb-resistant hardsuit may be a neat concept, but 99% of the time an EVA suit, which they already have, would work just fine.

I play RD all the time, and I don't think I've ever been killed by a bomb while wearing an EVA suit. If this was added, I'd probably wear it in the same situations I currently wear an EVA suit. Aside from the nicer graphic, it would make no difference. Not to me at least.

**Comment by Rumiluntti** [comment]
Science doesn't need anything extra, doubly so for RD.
Fuck no.

**Comment by SamHPurp** [comment]
We need less space suits, not more. RD needs no extra power.

RD can go on a bombing rampage even easier...

This is just all around a bad fucking idea.

**Comment by jughu** [comment]
What has this hard to do with making bomb hardsuits easier?

**Comment by Draconic-saint** [comment]
Devalues EVA even more, it's basically just a pinata for greyshirts right now. We need less of this catering to easy solutions. Not everything in this game is supposed to be easy.

So i'm saying no.

Sprite looks nice tho, i'd keep that one for !!FUN!! events, but i'm selfish like that.

File: code/modules/clothing/spacesuits/rig.dm [hunk 2]
```diff
@@ -473,6 +488,51 @@
...
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 100, bio = 100, rad = 60)
+	var/obj/machinery/doppler_array/integrated/bomb_radar
+	scan_reagents = 1
+	actions_types = list(/datum/action/item_action/toggle_helmet_light, /datum/action/item_action/toggle_research_scanner)
+
+/obj/item/clothing/head/helmet/space/rig/rd/New()
+	..()
+	bomb_radar = new /obj/machinery/doppler_array/integrated(src)
+
```
Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/rig.dm]: there needs to be a:

```
/obj/item/clothing/head/helmet/space/rig/rd/Destroy()
	if(bomb_radar)
		qdel(bomb_radar)
		bomb_radar = null
	return ..()
```

**Comment by DesolateG** [comment]
Wait, so this suit also has the same fire resistance as atmos, advanced, and ert hardsuit?

Not only no, but he'll no. RD doesn't need this.

**Comment by jughu** [comment]
@desolateG the suit itself is fireproof yeah your right .
But the helmet isn't plus they can make fireproof potions anyways so eh

**Comment by pinatacolada** [comment]
the helmet is also fire proof https://github.com/ParadiseSS13/Paradise/pull/6546/files#diff-7f322069c94cf98493bcb76467af5ba9R500
slime potions are late game, this is bomb space fire and mostly radiation proof _ROUND START_

also arguing that RD has suit access round start from EVA is equivalent to saying the CMO can have the hand teleporter because he already has access to it. EVA is meant to be a general emergency access thing, this suit however is a personal toy designed exclusively to the RD, having it starting in his office entitles the RD to do whatever he wants with it.

**Comment by jughu** [comment]
Emergency firesuit protection = fireproof? Nuh uh it isnt

**Comment by Taaketa** [comment]
RD doesn't need this at all. Just because his departments can blow, it's his job to STOP them from blowing up, not chill out in them after they have.

**Comment by Anoonki** [comment]
>when are hardsuits going to be slowed down indoors?

Have you ever worn a hardsuit indoors?
Have you ever wondered why you see engineers/atmos techs walking around in magboots carrying their hardsuits?

They're slow as fuck.
I acknowledge that RD is already overpowered, and probably doesn't need this, but I really would like to see this in the game.

**Comment by jughu** [comment]
🤔

**Comment by Necaladun** [comment]
Completely against giving more power to the RD.

The HoS doesn't have their own hardsuit, and might actually need one. 

People have been calling for science to be nerfed for a long time. Giving them more power is going quite against these wishes.

**Comment by davidchan13** [comment]
Why does every one assume the HoS needs a hardsuit? That's literally the Pod Pilot's job. AND Security already has hardsuits in the armory, they don't need more.

**Comment by Necaladun** [comment]
I don't think they need one - but the HoS's job actually might require one. There's nothing about the RD that does.

Also, with currently 13 thumbsup and 13 thumbsdown, this seems the very definition of controversial, so should be tagged as such when someone gets the chance.

**Comment by Anasari** [comment]
This is a substantial buff to RD antag who want to go into space. Given that right now, if the RD wish to go into space (as an antag), they'll have to use the shitty EVA suit, where they are likely to be seen and questioned by another crewmember beforehand, or steal a security or medical hardsuit that would require efforts on their part.

However, this PR simply hands the hardsuit over to them in a silver platter, and on top of that, this hardsuit also come with fire and bomb protection. I consider this a substantial buff to an already overpowered role of an overpowered department.

We do not need another buff when everyone is complaining about RD antag powergaming every round.

**Comment by Necaladun** [comment]
Good point Ansari - this seems to be a buff to antag RD's (Often said to be one of the most powerful antags already), and of dubious legit usefulness.

**Comment by DesolateG** [comment]
@Necaladun that's what I've been saying

**Comment by SamHPurp** [comment]
Currently antag RDs go and get the NASA suit from T-Comms and nobody ever questions them.

Do not merge this suit in, its a very bad idea for balance.

**Comment by Necaladun** [comment]
The sprite is great, though.

**Comment by Kyep** [comment]
If a RD floats around in space in an EVA suit, they're quite likely to get mistaken for a random joe, especially with no ID on. Consequences are likely to be small, if any at all.

If the RD does the same thing using this suit, they'll attract attention. Heads aren't generally allowed to leave the station, and the RD floating around in space in their suit will attract attention, and public radio comments, just like the Captain wearing their suit does. It is innately suspicious behavior.

I find it rather silly to suggest that this suit is going to help antag RDs. If anything, I think it will make them mind-numbingly obvious, and more easily caught.

Also, the idea that people will give the RD flak for getting an EVA suit is silly. I play RD a lot, and I regularly get an EVA suit, either because I'm an antag and need it, or because something blew up and I don't like pressure damage. I have NEVER, NOT EVEN ONCE been questioned as to why I am getting an EVA suit as RD. Heck, I don't even remember the last time I played scientist with an EVA suit and got questioned about that. People are used to seeing EVA suits. They don't care.
Walking around in the unique RD suit, though, especially outside of the station, will probably be treated much like the Captain walking around in their suit. Innately suspicious, and grounds for some alarm.

I'm still neutral on whether this PR gets merged or not. I don't think it matters, either way. 

However, I do feel like a lot of people are refusing to actually consider the mechanics impact of this change, and instead are allowing their opinions to be determined solely by their attitudes towards science in general. The anti-science people hate this PR. The regular science players like it. Very few people are actually forming an opinion based on the mechanical impact of the changes themselves. Which sucks. This PR could do with a lot more consideration based on its merits, and actual impact... and a lot less knee-jerk politics.

**Comment by DesolateG** [comment]
@keep brb I'm RD with hijack shuttle. . Gotta firebox the shuttle and escape real fast. It's ok, my hardsuit that people are desensitized to due to it becoming normal for RD to wear this cool looking suit will keep me from dieing to the flames.

**Comment by jughu** [comment]
>THIS BUFFS RD ANTAG>they are forced to get to eve for a suit... its just like  syndie soft/hardsuit doesnt exist

**Comment by jughu** [comment]
@desolateG its only has fire resistance of a emergency firesuit long exposure kills you too
Firepoof is a var for it nor burning it on a fire tile

**Comment by Anasari** [comment]
@jughu Syndie suit costs TC. This one's free.

This also does not match with the RD's role. I do not believe that RD have any business wandering around in space playing chicken with toxin bomb and walking all over plasma dust grenade explosion.

**Comment by SigholtStarsong** [comment]
Please, no. This doesn't fit the RD's role at all, and it blatantly encourages powergame. 

Give it to Sec instead of the standard bomb suit and it might be acceptable, maybe.

**Comment by Draconic-saint** [comment]
@Kyep How is adding a standard issue hardsuit, that's WAY more effective at protecting you than your standard EVA softsuit, not a buff to whoever that receives it? 

You increase the arsenal of the RD, the posibilities of whatever he can do, and you decrease the effort he has to take to prepare himself. It's not a direct buff like say removing the toxins bombcap, but a buff nonetheless.

And the flak this PR get's isn't because it allows the RD to acces a suit, everyone can acces a suit. It's that he get's one as part of his STANDARD gear. EVA gear should NOT be standard except for roles who's main job involves leaving the station boundaries. -> Pilots, engineers, miners. The RD, and scienists in general, do not fall under this category.

There's been calls to give the HoS a suit too, but i'm going to oppose those for the same reason. it's not their job to leave the station. If they NEED enviromental protection, they can go the EVA storage where **situational** suits are stored.

**Comment by Necaladun** [comment]
The anti-science attitude is general, from what I've heard, is due to the continuing power creep of science with no nerfs in sight. It's a big part of why many players have talked to me feeling like they're being ignored. 

Kyet is right that it isn't especially powerful by itself. But power creep is a big problem in science, and is based on a sum of the parts rather than any one individual thing.

**Comment by DesolateG** [comment]
Is it a cool suit? Yes, and I don't agree with making it admin only either.

Like I said on discord, it could be made as part of the equipment for a new ERT class.

**Comment by Rurik123** [comment]
I like the idea and sprite, but the RD really doesn't need it. If it were a suit that was admin spawnable for perhaps a event of some sort, then I do not see why not.

**Comment by Dinarzad** [comment]
Let's even evaluate this from outside of an "But muh antag RD" perspective for a moment.

What purpose does a non-shitter (No powergame, basically the ideal candidate that an RD is meant to be) RD have for a bomb-grade hardsuit?
He doesn't go out into space as any part of his job (Keeping an eye on his minions to make sure they don't go fuckin crazy or melt the station)
He might interact with the occasional Toxin bomb, and there is, conveniently, normal bomb suits already available in science for that reason.
And he doesn't diddle with super explosive things outside of toxins, because that's station damage.

This suit, is basically tailor made for antag RDs and nothing else. Because unless an RD is powergaming (Which is a thing we're trying to discourage), and is not an antagonist/bombing people an trying to survive them, he has, literally, no functional purpose for this piece of equipment. 
So the only reason he would have FOR this suit is to either powergame, or to be an antagonist. And the RD does not need -any- help in that field right now.

The sprite is really cool. The ITEM is really cool, I'd like for it to be in the code. But -not- like this.

**Comment by Kyep** [comment]
@DesolateG If a RD wears a hardsuit all the time for no reason, they're a comdom. Just like Captains who wear their hardsuit all the time for no reason, are comdoms. If this suit is merged, I'd actually support a new SOP item for RDs, saying they should not be wearing the hardsuit unless they have an actual reason to do so. Also, I can't remember the last time I saw an RD die to flames.

@Ansari One could argue that as the head of the department most likely to explode, routinely working with very explosive things, the RD does have legitimate business having an item that reduces the chances of them being destroyed by explosions. Just like regular scientists have bomb suits. RDs who wander around a bunch in space for no reason can and should be demoted simply for neglecting their job. Regardless of the clothes they are wearing while they do it. Equally though, I stand by my point that it isn't *necessary*. The RD does not *need* this. But, equally, I don't think it will matter much if they get it. Either way, not a big deal.

@Draconic-saint Because it doesn't actually matter whether you wear this, or an EVA suit. I play RD all the time, and I cannot remember a single case where I was wearing an EVA suit, and died, but, if I had been wearing this, I would have lived. Nor can I remember a case where another RD was in a similar situation. Is this mechanically stronger than an EVA suit? Sure. It has better numbers But does that difference actually have consequences? Does it result in a different *game experience*? I'm not convinced it does. You say it reduces the effort that the RD has to take to prepare themselves. That is true, but the effort to get an EVA suit is already extremely low - it just requires walking over and picking it up. Nobody questions it, you don't have to ask anyone. And you can just store it in your office afterwards. Heck, some people pick one up routinely as they walk by EVA, just in case they might use it later, so it isn't even out of the way. They don't even have to make a special trip over there! I would actually agree that the HoS should probably get a suit too. There are many situations (blobs, bomb blasts, suspects fleeing to space when there is no pod pilot, etc) where they need a suit, and unlike the RD, their suit actually needs additional protection beyond that of an EVA suit. Sure, they could use the armory suits, but those are already in short supply. More generally, I think there should be more incentives for good players to play Security in general, so I'd support giving the HoS something cool on those grounds.

@Necaladun I am skeptical that "power creep" is the problem with science. Every single major change to science I can remember since I started playing has been a fairly hard nerf. Genetics, for example, was nerfed pretty much into oblivion by the addition of genetic stability, which resulted in superpowers being quite rare now. The cargo-to-R&D mineral pipeline was reduced dramatically when upgraded components no longer made construction cheaper. Now, you're a lot less likely to see science pump out large volumes of stuff. Telescience lab was removed. Robotics lost access to protolathe. Just to give a few examples. Sure, some minor stuff has been added, like the transference potion, and quantum teleporters, but those require work to get, and in no way compensate for the hard nerfs to the major science sub-departments.
Science has got much LESS powerful since I started playing SS13. Yet, people still regularly call for nerfing it. It is as if no amount of nerfs will placate the people who hate science. I suspect that the problem isn't actually science itself. The problem is the attitude of certain people who play science - their attitude angers other players. Engineering can easily powergame by running around in full hardsuits, with cable cuffs, and the components to instantly make a stunprod, on them at all times. This makes them surprisingly dangerous. Security can easily powergame by searching *literally everyone* on blue or higher. Just set people to 'released' after search, and over time, you'll search the whole crew. Medical have several ways they can powergame with the cloner, health huds, etc. Yet, nobody complains about Eng/Sec/Medbay powergaming. It isn't because they can't powergame. It isn't even because they don't. I've seen them do it. "Shitcurity!" is a regular rallying cry, but NOBODY seriously advocates nerfing security when sec abuses the tools they have. Yet, that's what happens when scientists abuse science tools. Science is held to a different standard than the other departments. It has become popular to scapegoat science. There are a few people who hardcore powergame, and often, those people choose to play science. Rather than blaming those few individuals, though, we tend to blame the department as a whole. Job bans for bad security officers are common. Job bans for bad scientists are very rare. We can nerf whatever powergamers use, over and over, but if we nerf it too much they'll just switch to other jobs, and probably powergame just as hard, just in a different way, in another job. We're removing their tools without ever actually telling them to knock it off. Sometimes I wonder if all this talk of science being "too powerful" actually just makes the problem worse, because it *encourages* newer players who want power to play science, thus encouraging future powergaming, and more complaints. It furthers the perception that if you want power, play science. That's not a good perception, or what science should be about.
IMHO, the biggest imbalance between science and the other departments is that science jobs are, in general, more fun to play. Unlike cargo tech, atmos tech, etc they have a variety of things they can do in the round, and these things can actually be quite useful. Unlike engineer, medical doctor, etc, they have the ability to choose one of several quite diverse areas to focus on - and switch between them whenever they feel like it. Unlike engineer, medical doctor, etc, they have no official responsibilities. Short of blowing up their lab, or every single one of them failing to do R&D, nobody is going to yell at them for "not doing anything". If I had to distill the distinction, I'd say science jobs feel like playing for fun, and some other jobs feel like real work - and not in a good way. If we want to rebalance science compared to the other departments, IMHO we should start off with taking a hard look at other jobs which are either useless (e.g: atmos tech), simply not fun (cargo tech), or highly stressful (medical doctor). I'd rather see changes that increaes the overall fun of the game, by targeting those jobs that fall below the fun average, rather than looking to remove fun from the department that many see as the most fun/interesting.

With regards to this PR, my POV is: there's no good reason for the RD to get this suit (its unnecessary, the biggest pro in its favor is that it has a cool sprite), but equally, there is no good reason for them NOT to get the suit (it isn't a substantial power boost, and I actually suspect it will act as a slight nerf because the mechanical gains of wearing the suit are far outweighed by the additional scrutiny anyone wearing the suit will attract). 
It pretty much boils down to: do you feel like we need a good reason to merge this, or do we need a good reason not to merge it?

**Comment by uraniummeltdown** [comment]
won't be able to fix conflIct for a few days

**Comment by Draconic-saint** [comment]
@kyep you underestimate what it means to have to walk around. And it does in fact influence game experience. The difference between having to get a EVA suit and getting one standard is huge in regard to subtext & expectations of the job.

You say that there's no reason not to give it, but keep in mind that EVA activity is NOT a part of the RD's job. He maybe occasionally needs it because of circumstances. a occasional circumstance does not warrent his very own EVA suit, IMO.

**Comment by DesolateG** [comment]
@kyep we already have certain  RDs constantly running around in fully equipped war mechs and no one bats an eye, this will be no different.

**Comment by jughu** [comment]
Isn't that against SoP anyways

**Comment by Doukhan** [comment]
This suit and more suits in general would be fine IMO if hardsuits were slow as a one legged gazelle inside a pressurised environment, like you'd expect one to be.

**Comment by Da-Dman234** [comment]
Kyep has excellently brought up any points I'd like to make, so please take the time to read his posts as they are incredibly well thought out. I understand science powercreep is very much a real thing, but if anyone could give an example as to how exactly this would benefit an antag RD I would like to hear it. This is an easily identifiable EVA suit that doesn't do much other than add some very basic resistances that are helpful to the RD in dealing with inevitable messes players in science cause. As has been brought up, this doesn't make you immune to explosives, and the only thing I see that should be even somewhat controversial is the fact that is provides _some_ fire protection. This really seems like a kneejerk reaction simply because it's science related, and I hope you all reconsider.

**Comment by SigholtStarsong** [comment]
Changeling antags could simply carry a Dead Man's Trigger bomb, get caught
out, be beaten to death and all of the sudden all of Sec is now dead.

Rejuvenate, free suicide maxcap, barely any consequences.

**Comment by jughu** [comment]
Well i mean you could already do that in a ce hardsuit with 50% chance a coinflip

**Comment by SigholtStarsong** [comment]
CE doesn't have immediate access to easy maxcap bombs. He'd have to
specifically construct facilities for it.

Plus, it's a coinflip - 50% of the time he will be gibbed. That's a huge
quantity of risk versus reward.

**Comment by Dinarzad** [comment]
>understand science powercreep is very much a real thing, but if anyone could give an example as to how exactly this would benefit an antag RD I would like to hear it.

No. Not until someone finally explains to me why a non-antag, non-powergaming RD needs this suit first. Because right now there's a reason for him not to have it. (Encourages powergaming behaviour even more and gives Antag RD's a free super hardsuit) and there is NO reason for him TO have it.

The RD's job is to manage his department not play Ultra Man on the station.
He should be on the station making sure Robotics isn't making death mechs and handing them out, making sure RnD isn't giving everyone night-vision secHUDS or X-ray lasers on a whim.
Making sure Xenobio isn't breeding an army worthy of Mordor, and that Toxins isn't leaving high explosives just chilling on a table and are giving them to mining.
None of those involve space, defusing bombs or wading into fires.

If something explodes in Science, it is no longer his job, but the job of the CE to fix.
If someone starts a plasma fire it is NOT his job to fix, it is the job of Atmospherics and the CE to play Firefighter.
The RD's job is on the station, never in space, so he doesn't even need a softsuit, let alone a hardsuit with bomb and fire protection.

So until someone tells me -WHY- the RD needs STANDARD access to a hardsuit, then I am NOT going to do anything other then vehemently oppose this, because a "1 in 100" chance he might need it, is a reason to go into EVA and grab a suit, not have one made standard in his loadout.

**Comment by Da-Dman234** [comment]
> Changeling antags could simply carry a Dead Man's Trigger bomb, get caught
out, be beaten to death and all of the sudden all of Sec is now dead.
Rejuvenate, free suicide maxcap, barely any consequences.

Except for the inevitable bwoinking he's going to get for using a maxcap. Bombs are pretty restricted on our server, if someone chooses to abuse the suit with toxins I'd only assume they'd get a ban. As far as I know the only people who can get escape alone are traitors, so with the exception of the rare roll of a Traitor Changling RD, I don't see this being a problem.

> No. Not until someone finally explains to me why a non-antag, non-powergaming RD needs this suit first. Because right now there's a reason for him not to have it. (Encourages powergaming behaviour even more and gives Antag RD's a free hardsuit) and there is NO reason for him TO have it.
 The RD's job is to manage his department not play Ultra Man on the station.
He should be on the station making sure Robotics isn't making death mechs and handing them out, making sure RnD isn't giving everyone night-vision secHUDS or X-ray lasers on a whim.
Making sure Xenobio isn't breeding an army worthy of Mordor, and that Toxins isn't leaving high explosives just chilling on a table and are giving them to mining.
None of those involve space, defusing bombs or wading into fires.
If something explodes in Science, it is no longer his job, but the job of the CE to fix.
If someone starts a plasma fire it is NOT his job to fix, it is the job of Atmospherics and the CE to play Firefighter.
The RD's job is on the station, never in space, so he doesn't even need a softsuit, let alone a hardsuit with bomb and fire protection.
So until someone tells me -WHY- the RD needs STANDARD access to a hardsuit, then I am NOT going to do anything other then vehemently oppose this, because a "1 in 100" chance he might need it, is a reason to go into EVA and grab a suit, not have one made standard in his loadout.

Because science is inherently a dangerous department? Even more so now that it's filled with newbies. I don't know where you're getting the impression that RD is the ultraman on station, because he still lacks many of the materials needed for actual construction (Piping, Door boards, ect.), and while everything you listed is very much part of his job, it also entails keeping science safe and livable in general. This suit would allow the RD to do basic patches so science is atleast SOMEWHAT usuable after an accident. 

Your idea that the CE and Atmos Techs play firefighter is laughable for multiple reasons (mainly involving our atmos system and the fact that they go out before anyone more than a minute away can reach them), and as I pointed out before the RD will still be unable to do actual legitimate construction outside of basic patchwork.

Also, I'm trying saying this in the kindest way, but your wording and attitude very much screams of a kneejerk reaction to me. If it were any other department than science getting a cool toy I doubt you'd be "vehemently opposed". As for why science needs this, they don't, but I'm always for adding more cool and unique items to any department, as well as the fact that I see this being moderately helpful for regular RDs, and pretty useless for antags.

**Comment by DesolateG** [comment]
If you see this suit as useless, then you're either trolling or blind. RD doesn't need this suit.

**Comment by Da-Dman234** [comment]
> If you see this suit as useless, then you're either trolling or blind. RD doesn't need this suit.

Nope, neither. Just like new stuff added to the game, especially when there really isn't any reason not to.

**Comment by DesolateG** [comment]
We have listed very valid reasonso why it shouldn't be.

**Comment by Da-Dman234** [comment]
> If you see this suit as useless, then you're either trolling or blind. RD doesn't need this suit.

Yes, I said he didn't _need_ it, but it would certainly make their job easier.

**Comment by DesolateG** [comment]
Yup, it certainly will make an antag RDs job easier

**Comment by SigholtStarsong** [comment]
He not only doesn't NEED it, he shouldn't be anywhere NEAR it.

**Comment by Da-Dman234** [comment]
I have yet to see a valid argument as to how this could benefit a traitor RD other than the fire resistance, of which is does not even block fully. If that's really such a problem then just remove/significantly lower it. There's really no point in restating arguments that have already been made above.

**Comment by SigholtStarsong** [comment]
As I have repeatedly stated in this very thread, this is hilariously
abusable by changeling antagonists or hell, even lich wizards.

**Comment by DesolateG** [comment]
If you have yet to seen a "valid argument against it" despite what we have posted then you either A. Don't play antag enough or B. Refuse to listen to what we have said.

**Comment by taukausanake** [comment]
I mean I still like it but don't want the RD to get it. Research is still my answer even though it will still end up in the hands of Science and by extension the RD. It's too interesting to not have but too dangerous to have.
You really can't just give it a high research cost and materials to make? I still know it wouldn't matter but you can't just ignore this

**Comment by Da-Dman234** [comment]
And as I said earlier, fairly sure Changlings would eat a ban, and Wizard Lich? Really? Yeah, let's just specifically hunt down the RD for his suit, and then take a 10 minute break in toxins.

You're nitpicking my words and choosing what's convienent for you here. I understand the argument for powercreep, and EVA, but legitimately not a single valid example for how an antag can abuse it has been given, minus your firebomb scenario (Which he isn't even fully immune to), which is easily remedied if it proves to be a problem.

As I stated before, Kyep has hammered across all the points I'd like to make as for why I'd like this to happen, and I'll leave it at that.

**Comment by SigholtStarsong** [comment]
And you're intentionally ignoring points made by others in this thread.

The RD does not and will not have any buisness doing anything that this
suit is designed for, and if he IS doing stuff like what this suit enables,
he should be bwoinked almost immediately.

**Comment by DesolateG** [comment]
And we have debunked every single one of those reasons. Bottom line, that's not the RD's job.

Adding something just because it's "neat" is a bad idea all around and not a valid reason either when it comes to something like this.

**Comment by Da-Dman234** [comment]
I fail to see how doing basic patches when his workstation inevitably blows up isn't part of his business. Neither of you have addressed the fact that he cannot make actual repairs outside of basic station patching.

**Comment by DesolateG** [comment]
That's not his job. Station gets breached then you evacuate that area and call engineering, because that IS their job.

**Comment by Da-Dman234** [comment]
True, but I would argue that it's the RD's to keep his workstation in some semblance of order and repair. With how frequently science blows up because of one mistake, and engineers sometimes less than ideal response time, I don't think it's unfair to allow the RD to easily patch up his workstation so work can continue, and an entire department full of people aren't out of work.

**Comment by DesolateG** [comment]
Again, that is not the RD's job. His job is to direct research  (as the title implies) and PREVENT science from damaging the station.

**Comment by Da-Dman234** [comment]
It is the RD's job to keep his workplace functional. There is no proactive way to prevent toxins and chem from causing damage minus just outright disabling it, only reactive ways, one of which is to fix the hole so people can resume work. I just personally witnessed science blow up and Engineering taking 30+ minutes to respond when the explosion rendered both Toxins and the chemlab unusable. An entire department should not be out of work for an extended amount of time until someone from science was bothered to grab an EVA suit (which nobody questioned by the way). Science exploding is a frequent occurrence and it's reasonable to allow the RD to easily do basic patches.

**Comment by SigholtStarsong** [comment]
If it's an "easy, basic patch", then why would he even need a hardsuit?

**Comment by DesolateG** [comment]
If toxins blows up then the RD is to blame for not proactively coaching their scientist on how to safely handle plasma. If it's not fixed then the CE and engineering are to blame.

There's this neat thing called Command channel, use it.

**Comment by Dinarzad** [comment]
> Also, I'm trying saying this in the kindest way, but your wording and attitude very much screams of a kneejerk reaction to me. If it were any other department than science getting a cool toy I doubt you'd be "vehemently opposed".

You're right. I wouldn't oppose it as much. ('As much' being the key word, as it depends on WHAT Department would be given this.)
Because other departments don't have what science already has, and the RD job is already one of the most abused job slot for antag/powergaming.
And if you can't take any tone of disagreement as anything other than "A kneejerk reaction" then you're not in a position to sensibly argue, logically, why the position that has access to Memechems, Death Mechs, and every single widget and gun the Protolathe can spit out (OR almost every, if it's not an antag with an E-mag) also needs a free fire and explosive resistant hardsuit.
Especially when that suit adds -nothing- to the RD's job description.

Hull patching is NOT his job, you call engineers for that. If engineers don't come, then you bitch the CE out of command or go over his head. If they STILL don;t come then steal a soft suit from EVA and THEN do it yourself, but none of that is exactly 'standard'.
You may consider it laughable to consider Atmos and CE to be Firefighters, but that is very much their job description, considering Atmos has a Fire Axe, special fire suits and an extinguisher backpack for just that reason. Fires going out before they get there isn't a reason to buff the RD, it;s a reason to buff FIRES if anything.
RD should get that hardsuit the same time the CMO should get a E-Gun to help fend off antags and greytide that enter Medbay to keep it safe.

**Comment by uraniummeltdown** [comment]
@Crazylemon64 please restart Travis

**Comment by adamkad1** [comment]
it would be legit if it was high level, high cost research item

**Comment by DesolateG** [comment]
Researchable hardsuits are never legit

**Comment by Crazylemon64** [comment]
Mass-producable hardsuits sound like a very bad idea

**Comment by Rumiluntti** [comment]
Trying to balance anything with research levels is not legit

**Comment by Spacemanspark** [comment]
Remember power armor?
Never again.

**Comment by taukausanake** [comment]
I just thought of something. You know how we're supposed to have Traders that visit the station and trade stuff? Why not make this one of the things they can trade with? Why not give them a bunch of weird shit that has no good reason to be on the station except that it _could_ be if the crew wants it bad enough? I mean, it's not like that event is any good anyways.
OR maybe make it a special item in an away mission that you have to fight through. I good reward for such possible sacrifice, if you can call it that in any of the away missions we have.

That moment you realize Paradise needs way more work than what is currently being done for it

**Comment by jughu** [comment]
Gatewsy mission... involving bombs defusing to progress something something...

**Comment by Crazylemon64** [comment]
The heads and I are in agreement that this is something that the RD should not have, so closarino it is

**Comment by uraniummeltdown** [comment]
@Crazylemon64 whoa whoa whoa we can talk about this, please reopen
How about adding it instead to the following places: derelict research area on z4 and the rd lockers in the gateway missions

**Comment by tigercat2000** [comment]
@uraniummeltdown It would be better if you opened a new PR to allow for it to be properly discussed under those terms. It becomes messy quickly if a significantly controversial PR completely changes what it does after a few days.

**Pull request closed without merging**